README.md

```diff
@@ -22,7 +22,7 @@ Dragonfly is now hosted by the Cloud Native Computing Foundation (CNCF) as an In
 
 ## Installation
 
-Please follow [this chart document](./charts/dragonfly/README.md)
```

Comment:
Where is the `./charts/dragonfly/README.md` still referenced?

Reply:
It will referenced in https://artifacthub.io/packages/helm/dragonfly/dragonfly